app/controllers/concerns/blacklight/default_component_configuration.rb

```diff
@@ -64,39 +65,26 @@ def add_show_tools_partial name, opts = {}
         end
       end
 
-      ##
       # Add a tool to be displayed for each document in the search results.
-      # @param partial [String] the name of the document partial
-      # @param opts [Hash]
-      # @option opts [Symbol,Proc] :if render this action if the method identified by the symbol or the proc evaluates to true.
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      # @option opts [Symbol,Proc] :unless render this action unless the method identified by the symbol or the proc evaluates to true
-      #                             The proc will receive the action configuration and the document or documents for the action.
-      def add_results_document_tool name, opts = {}
+      # @see #add_nav_action for params
```

Comment:
As a community we are at best inconsistent about producing yard/rdoc that is worth reading, throughout the stack.  I only read the docs on github (instead of rubydoc et al.), because it allows me to check immediately whether they are bogus or not.  These sections of blacklight are better than most (so kudos for that), but still have factual errors (e.g., parameter name is `name`, not `partial`).  

I'm amenable to using a macro, but to me that's less transparent than the `@see` statement, largely directed at an output I don't consume.  It does have the benefit of being visible at the point of concentration, with the downside of being essentially a novel piece of code to worry about.